src/explore-education-statistics-admin/src/pages/release/pre-release/ReleasePreReleaseAccessPage.tsx

```diff
@@ -85,29 +80,10 @@ const ReleasePreReleaseAccessPage = () => {
               </>
             )}
 
-            {release.approvalStatus === 'Approved' ? (
-              <PreReleaseUserAccessForm
-                releaseId={release.id}
-                isReleaseLive={release.live}
-              />
-            ) : (
-              <WarningMessage>
```

Comment:
I take it we don't have any Robot tests looking for this text currently?

Reply:
I cannot find anything in the UI tests. I ran the prerelease.robot test suite and everything passed.